account_banking_pain_base/models/res_partner_bank.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Brain-Tec Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+
+
+class ResPartnerBank(models.Model):
+    _inherit = 'res.partner.bank'
+
+    @api.model
+    def _default_account_holder(self):
+        return self.partner_id.name
```

Comment:
Would it make sense to leave it empty by default?

Reply:
This here should be removed as it is not working anyway. The default is set in views/res_partner.xml by default_account_holder in context.